src/Yoti/YotiClient.php

```diff
@@ -52,50 +60,63 @@ class YotiClient
     private $_mockRequests = false;
 
     /**
+     * @var string
+     */
+    private $_yotiSdkHeader;
```

Comment:
Other SDKs are using the name `sdkIdentifier`, up to you if you want to use the same.

Reply:
yes I will use the sdkIdentifier instead to keep consistency.